src/golang/cmd/server/handler/edit_dynamic_engine.go

```diff
@@ -55,9 +55,12 @@ func (*EditDynamicEngineHandler) Headers() []string {
 	}
 }
 
+type dynamicEngineAction string
+
 const (
-	createAction string = "create"
-	deleteAction string = "delete"
+	createAction      dynamicEngineAction = "create"
+	deleteAction      dynamicEngineAction = "delete"
+	forceDeleteAction dynamicEngineAction = "force-delete"
```

Comment:
nit: Can you note that this reflects `K8sClusterActionType` in Python and that they should be kept in sync?